### ISSUE TITLE
Icon Button Tooltip Not Accessible Via Keyboard

### ISSUE THREAD

**Issue by bmeersma** [OfficeDev/office-ui-fabric-react#12026]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: Edge Version 81.0.416.11 on Windows 10

### Describe the issue:

Keyboard users cannot access tooltips associated with the icon button control

#### Actual behavior:

A tooltip is available on hover of the icon button control, but the tooltip is not accessible to keyboard users.

#### Expected behavior:

Keyboard users are able to view the tooltip associated with an icon button.

**Comment by aneeshack4** [comment]
This is a dupe of #11298 - default HTML behavior dictates that tooltips don't show on keyboard focus.
